src/usecase.js

```diff
@@ -85,6 +91,33 @@ class UseCase {
     }
 
     get auditTrail() {
+
+        if(this._mainStep.auditTrail.configuration && this._mainStep.auditTrail.configuration?.mode === "minimal")
```

Comment:
Since is is used so many times, `this._mainStep.auditTrail.configuration` can be stored in a variable, for a cleaner code